src/DssDirectDepositHub.sol

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-pragma solidity 0.6.12;
+pragma solidity >=0.6.12;
```

Comment:
Keeping the pragma this way is actually a bug. As compiling with solc 0.6.x will make the safe math is not applied and then the code has a complete different behaviour. Not sure if 0.7.0 is already implementing it but to be honest I'd just do:
```
pragma solidity ^0.8.13;
```

Reply:
Yes strongly agree with @gbalabasquer here.